test/phpunit/AdminLibTest.php

```diff
@@ -48,7 +49,23 @@
  */
 class AdminLibTest extends CommonClassTest
 {
-	protected $backupGlobalsBlacklist = array('conf', 'user', 'langs', 'db');
```

Comment:
I don't see reason of having this line of code.
May be we can remove it completely ?

Reply:
[I can only think that the author of the test ](https://docs.phpunit.de/en/9.6/fixtures.html?highlight=backupGlobalsExcludeList#global-state)

I think that excluding the variables from backup/restore implies that if the test modifies the values that their values are maintained.

This is the commit where it was introduced:
https://github.com/Dolibarr/dolibarr/commit/599aa6009a6